vendors/espressif/boards/esp32/aws_tests/application_code/espressif_code/amazon-freertos-tests/component.mk

```diff
@@ -52,7 +52,7 @@ COMPONENT_SRCDIRS := $(AMAZON_FREERTOS_SDK_DIR)/standard/mqtt/src \
         $(AMAZON_FREERTOS_ABSTRACTIONS_DIR)/pkcs11/mbedtls \
         $(AMAZON_FREERTOS_ABSTRACTIONS_DIR)/platform/freertos \
         $(AMAZON_FREERTOS_ABSTRACTIONS_DIR)/secure_sockets/freertos_plus_tcp \
-        $(AMAZON_FREERTOS_3RD_PARTY_DIR)/http-parser \
+        $(AMAZON_FREERTOS_3RD_PARTY_DIR)/http_parser \
```

Comment:
We probably need to do this for the demo project also

Reply:
ok let me check

Reply:
do a global search and replace in vendors directory